src/views/Welcome/Welcome.tsx

```diff
@@ -1,5 +1,64 @@
-import React from 'react';
+import React, {useCallback} from 'react';
+import {useNavigate} from 'react-router-dom';
+import {useGame} from '@contexts/game.context';
+import {MAX_PLAYERS, PlayerColorBank} from '@components/app/consts';
+import openings from '@assets/openings.json';
+
+const cetegories = Object.keys(openings);
 
 export const Welcome = (): React.JSX.Element => {
-    return <h1>Welcome</h1>;
+    const navigate = useNavigate();
+    const { config: { players, openerCategory }, setConfig } = useGame();
+    const inputRef = React.useRef<HTMLInputElement>(null);
+
+    const onCategoryChange = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
+        setConfig((prevState) => ({...prevState, openerCategory: event.target.value as Game['openerCategory']}));
+    }, [setConfig]);
+
+    const onPlayClick = useCallback(() => {
+        const isValid = players.every((player) => player.name.trim().length);
+        if (isValid) {
+            navigate('/');
+        }
+    }, [players, navigate]);
+
+    const addPlayer = useCallback(() => {
+        const value = inputRef.current?.value;
+        if (value) {
+            setConfig((prevState) => ({
+                    ...prevState,
+                    players: [...prevState.players, { name: value, id: prevState.players.length + 1, color: PlayerColorBank.player3}]
```

Comment:
Several things 
1. should have mentioned it before , i think it makes sense here to have a max limit of 5 players to the game and enforce it at this stage, and a minimum of 2. 
2. all added players get the same player color it should be added dynamically.
3. For some reason from what i saw the third added players does not have a progress colored bar running.
4. I think we can remove the hard coded players and for development and debugging purposes (or perhaps also a good UI decision) have a quick play button - that can be extracted to a different ticket.

Reply:
Maybe we can add a param in the url - ?debug=true

Reply:
maybe,  or "quickplay" with player names player 1 player 2